views/releases/index.hbs

```diff
@@ -61,47 +50,47 @@ $ ./node_modules/.bin/electron .</code></pre>
       {{/if}}
 
       {{#each releasesPage.page.data}}
-        <h2 class="release-entry" id="{{this.data.version}}">
-          <a class="tag" href="#{{this.data.version}}">Electron {{this.data.version}}</a>
-          <em class="date" data-date="{{this.data.published_at}}" data-format="%B %d, %Y">{{this.data.published_at}}</em>
-          <em class="date ltr-only">(<span data-date="{{this.data.published_at}}"></span>)</em>
-          <a href="{{this.data.html_url}}" class="tag octicon-wrapper"><span class="octicon octicon-mark-github vertical-middle"></span></a>
-        </h2>
-
-        {{{this.data.body_html}}}
-
+        <article>
+          <h1 class="release-entry" id="{{this.data.version}}">
+            <a class="tag" href="#{{this.data.version}}">Electron {{this.data.version}}</a>
+            <em class="date" data-date="{{this.data.published_at}}" data-format="%B %d, %Y">{{this.data.published_at}}</em>
+            <em class="date ltr-only">(<span data-date="{{this.data.published_at}}"></span>)</em>
+            <a href="{{this.data.html_url}}" class="tag octicon-wrapper"><span class="octicon octicon-mark-github vertical-middle"></span></a>
+          </h1>
+          {{{this.data.body_html}}}
+        </article>
         <hr>
```

Comment:
Not sure our use case here applies for [thematic break](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/hr) now that we are using article 🤔

Reply:
That's a good point. I can remove that element as well.